hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CryptoTransferValidator.java

```diff
@@ -168,7 +168,7 @@ public void validateSemantics(
 
             // Verify that the current total number of (counted) fungible transfers does not exceed the limit
             validateTrue(
-                    totalFungibleTransfers < ledgerConfig.tokenTransfersMaxLen(),
+                    totalFungibleTransfers <= ledgerConfig.tokenTransfersMaxLen(),
```

Comment:
Was this just to match mono service?  Did this actually make a difference?

Reply:
Yes it makes a difference; we want to allow up to 10 adjustments in this list, so `<=` is necessary given the default property value.